### ISSUE TITLE
[FEATURE] Also show cartesian areas/perimeters in identify results

### ISSUE THREAD

**Issue by QGIS-feature-tracker** [qgis/QGIS-Documentation#2615]
Original commit: https://github.com/qgis/QGIS/commit/f9be6055375ccf60ec426e561c0f10e88613f63f by nyalldawson

Because users are still getting confused with the difference
between the cartesian areas and ellipsoidal areas, show both
in the identify results dock.

The reasoning here is:

- if a user understands this concept, they will know the correct
one to use, and its good for us to inform them of the difference
here. Plus, it means they immediately see if the ellipsoid
setting is correct and the difference it is making for the
area/length calculation.

- if a user has no idea what the difference is, then we should
make them aware that there's (at least!) two different possible
measurement values. They can then either research what these mean and make
the right choice (and become better informed GIS practitioners),
OR pick a random one - which really is no different then the
previous situation, because an uniformed user is just as likely
to be working in an unsuitable projection with a poor ellipsoid
choice.

In short, we don't try to guess the right choice for users
and instead give them all the information and let them make the
call which value to use.
